website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,10 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Add marketplace regional purchase indicator to captures and capture refunds
+The `marketplace` concept has been added to captures and capture refunds. The
+`marketplace[regional_purchase]` parameter is optional with default `true`.
```

Comment:
```suggestion
`marketplace[regional_purchase]` parameter is required if the merchant is a
marketplace.
```

Because we are considering this option.

Reply:
We should also add a timestamp for when the requirement is enforced.

Reply:
Should we run this by the @clearhaus/partner team to land on a date for enforcement?

Reply:
Yeah, let's do that :+1:

Reply:
I've asked internally and am awaiting answers :heavy_check_mark:

Reply:
For people following this: The requirement will be enforced for new marketplaces right away (when deployed).